features/webxr-depth-sensing.yml

```
@@ -0,0 +1,19 @@
+name: WebXR Depth Sensing
+description: WebXR Depth Sensing provides information about the distance from the user's device to the real-world geometry in the user's environment.
```

Again, not knowing enough about this API, but something along these lines, with an entry point, seems likely:

```suggestion
description: The [mumble mumble] API measures the distance from the user's device to the real-world geometry in the user's environment.
```

---

There are two entry points here. One is depth information from the CPU, and the other is from WebGL. They have a common parent interface, and maybe it is sufficient enough to mention that here? Otherwise, a sentence including entry points would be a bit longer.

```suggestion
description: The `XRDepthInformation` API measures the distance from the user's device to the real-world geometry in the user's environment.
```